### PR TITLE
4.7.19 release notes

### PR DESCRIPTION
**Pull request by agh1**

### CODE REVIEW

File: release-notes/4.7.19.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+- **[Miscellany](#misc)**
+- **[Credits](#credits)**
+
+## <a name="features"></a>Features
+
+### Core CiviCRM
+
+- **[CRM-20264](https://issues.civicrm.org/jira/browse/CRM-20264) Store CC type and last 4 digits from Contribution form ([10234](https://github.com/civicrm/civicrm-core/pull/10234), [10228](https://github.com/civicrm/civicrm-core/pull/10228), [10157](https://github.com/civicrm/civicrm-core/pull/10157), [10193](https://github.com/civicrm/civicrm-core/pull/10193), [10194](https://github.com/civicrm/civicrm-core/pull/10194), and [10189](https://github.com/civicrm/civicrm-core/pull/10189))**
+
+- **[CRM-20449](https://issues.civicrm.org/jira/browse/CRM-20449) Possible regression - enotices from 'send thankyou letter' on contribution search actions ([10219](https://github.com/civicrm/civicrm-core/pull/10219))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: bug

File: release-notes/4.7.19.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-19773](https://issues.civicrm.org/jira/browse/CRM-19773) Call hook_civicrm_selectWhereClause from the BAOs ([10125](https://github.com/civicrm/civicrm-core/pull/10125) and [9544](https://github.com/civicrm/civicrm-core/pull/9544))**
+
+- **[CRM-17711](https://issues.civicrm.org/jira/browse/CRM-17711) Add Support for Backdrop ([10184](https://github.com/civicrm/civicrm-core/pull/10184))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: I'm going to * a few I think should go near the top to highlight - incl this

File: release-notes/4.7.19.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-20417](https://issues.civicrm.org/jira/browse/CRM-20417) Translate strings ([10179](https://github.com/civicrm/civicrm-core/pull/10179) and [10148](https://github.com/civicrm/civicrm-core/pull/10148))**
+
+- **[CRM-20439](https://issues.civicrm.org/jira/browse/CRM-20439) Rename card_type to card_type_id of civicrm_financial_trxn table (IIDA-126) ([10178](https://github.com/civicrm/civicrm-core/pull/10178))**
+
+- **[CRM-20420](https://issues.civicrm.org/jira/browse/CRM-20420) Allow retention of generated PDFs/Docs  ([10155](https://github.com/civicrm/civicrm-core/pull/10155))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: *

File: release-notes/4.7.19.md [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-20312](https://issues.civicrm.org/jira/browse/CRM-20312) Identify and fix missing indexes ([10109](https://github.com/civicrm/civicrm-core/pull/10109), [10106](https://github.com/civicrm/civicrm-core/pull/10106), and [10108](https://github.com/civicrm/civicrm-core/pull/10108))**
+
+- **[CRM-20313](https://issues.civicrm.org/jira/browse/CRM-20313) Add index to civicrm_activity.status_id ([10169](https://github.com/civicrm/civicrm-core/pull/10169) and [10025](https://github.com/civicrm/civicrm-core/pull/10025))**
+
+- **[CRM-20384](https://issues.civicrm.org/jira/browse/CRM-20384) Bookkeeping report add contrib ID, filter by financial account owner (IIDA-85) ([10112](https://github.com/civicrm/civicrm-core/pull/10112))**
+
+- **[CRM-20391](https://issues.civicrm.org/jira/browse/CRM-20391) Add pan_truncation to search form ([10139](https://github.com/civicrm/civicrm-core/pull/10139))**
+
+- **[CRM-19813](https://issues.civicrm.org/jira/browse/CRM-19813) Hook priorities and core hooks to support LExIM ([10138](https://github.com/civicrm/civicrm-core/pull/10138) and [9949](https://github.com/civicrm/civicrm-core/pull/9949))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: *

File: release-notes/4.7.19.md [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-20400](https://issues.civicrm.org/jira/browse/CRM-20400) Update the Payment Receipt to use for Contribution pyament ([10128](https://github.com/civicrm/civicrm-core/pull/10128))**
+
+- **[CRM-20367](https://issues.civicrm.org/jira/browse/CRM-20367) Add wrapper function for adding and dropping Index in Upgrader ([10131](https://github.com/civicrm/civicrm-core/pull/10131) and [10086](https://github.com/civicrm/civicrm-core/pull/10086))**
+
+- **[CRM-20318](https://issues.civicrm.org/jira/browse/CRM-20318) Add is_public setting on Custom Groups. ([10028](https://github.com/civicrm/civicrm-core/pull/10028))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: half a star :-)

File: release-notes/4.7.19.md [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-16633](https://issues.civicrm.org/jira/browse/CRM-16633) Case Subject not Editable ([10123](https://github.com/civicrm/civicrm-core/pull/10123))**
+
+### CiviReport, NYSS
+
+- **[CRM-20368](https://issues.civicrm.org/jira/browse/CRM-20368) report templates: don't hard-code order bys ([10087](https://github.com/civicrm/civicrm-core/pull/10087))**
+
+### Dedupe
+
+- **[CRM-20328](https://issues.civicrm.org/jira/browse/CRM-20328) Remove code duplication when calling Dedupe functions ([10150](https://github.com/civicrm/civicrm-core/pull/10150), [10069](https://github.com/civicrm/civicrm-core/pull/10069), [10070](https://github.com/civicrm/civicrm-core/pull/10070), and [10064](https://github.com/civicrm/civicrm-core/pull/10064))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: the dedupe are all tidy ups towards later work

File: release-notes/4.7.19.md [hunk 7]
```diff
@@ -0,0 +1,325 @@
...
+
+- **Make sourceSQLFile throw an error when it can't find a file. ([10160](https://github.com/civicrm/civicrm-core/pull/10160))**
+
+- **[CRM-20429](https://issues.civicrm.org/jira/browse/CRM-20429) Deleting an activity with target contacts > 50 results in error ([10162](https://github.com/civicrm/civicrm-core/pull/10162))**
+
+- **[CRM-20339](https://issues.civicrm.org/jira/browse/CRM-20339) Custom field data on relationships only saved when creating a new relationship, not on edit ([10092](https://github.com/civicrm/civicrm-core/pull/10092))**
+
+- **Used Api to get financial account for an entity defined in civicrm_en… ([10130](https://github.com/civicrm/civicrm-core/pull/10130))**
+
+- **[CRM-20361](https://issues.civicrm.org/jira/browse/CRM-20361) Populate tokens when Scheduled Reminders are set to Record Activity for Automated Email ([10081](https://github.com/civicrm/civicrm-core/pull/10081))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: *

File: release-notes/4.7.19.md [hunk 8]
```diff
@@ -0,0 +1,325 @@
...
+
+- **[CRM-20350](https://issues.civicrm.org/jira/browse/CRM-20350) Incorrect escaping of double-quotes in iCalendar text values ([10067](https://github.com/civicrm/civicrm-core/pull/10067))**
+
+- **[CRM-20346](https://issues.civicrm.org/jira/browse/CRM-20346) Remove encryptDB.php from codebase ([10059](https://github.com/civicrm/civicrm-core/pull/10059))**
+
+- **[CRM-20311](https://issues.civicrm.org/jira/browse/CRM-20311) Membership cancelation via contribution doesn't create membership status change activity ([10054](https://github.com/civicrm/civicrm-core/pull/10054) and [10024](https://github.com/civicrm/civicrm-core/pull/10024))**
+
+- **(NFC) Update stale GenCode checksums ([10048](https://github.com/civicrm/civicrm-core/pull/10048))**
+
+- **[CRM-20011](https://issues.civicrm.org/jira/browse/CRM-20011) Profile date fields using YYYY-MM or MM YY don't work ([10003](https://github.com/civicrm/civicrm-core/pull/10003))**
```
Comment (eileenmcnaughton) [inline release-notes/4.7.19.md]: half a star :-)

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by totten** [comment]
@agh1 Is this still WIP?

**Comment by agh1** [comment]
@totten yes working on it actively--should be done tonight

**Comment by agh1** [comment]
@totten all ready to go

**Pull request merged**